scotty/fun_general.py

```diff
@@ -459,38 +489,58 @@ def find_epsilon_g(electron_density, B_Total, launch_angular_frequency):
 
 
 def find_Booker_alpha(
-    electron_density, B_Total, sin_theta_m_sq, launch_angular_frequency
+    electron_density,
+    B_Total,
+    sin_theta_m_sq,
+    launch_angular_frequency,
+    electron_mass=constants.m_e,
 ):
-    epsilon_para = find_epsilon_para(electron_density, launch_angular_frequency)
+    epsilon_para = find_epsilon_para(
```

Comment:
A separate issue, but this reminds me we should probably use `DielectricTensor` here to avoid recomputing these values -- let me do that in a separate PR though